src/validations/auth.js

```diff
@@ -4,48 +4,67 @@ export default {
   signupSchema: [
     check('firstName')
       .trim()
-      .exists().withMessage('First name is required')
+      .exists()
+      .withMessage('First name is required')
```

Comment:
Please help change to 
```suggestion
      .withMessage('first name is required')
```